clients/ethereumjs/Dockerfile.local

```diff
@@ -0,0 +1,35 @@
+### Build Ethereumjs Locally:
```

Comment:
this can be merged to the main Dockerfile using `RUN if` to either git clone repo or copy from local